Scripts/build-ios-framework.sh

```diff
@@ -49,7 +49,14 @@ cp -f "${SRCROOT}/${FMK_NAME}/Support/iOS.modulemap" "${INSTALL_DIR}/Modules/mod
 
 # Copies the headers and resources files to the final product folder.
 cp -R "${SRCROOT}/${WRK_DIR}/Release-iphoneos/include/${FMK_NAME}/" "${INSTALL_DIR}/Headers/"
+
+# Copy the resource bundle for App Center Distribute.
+if [[ "$FMK_NAME" == "AppCenterDistribute" ]]; then
+echo "Copying resource bundle for $FMK_NAME."
 cp -R "${SRCROOT}/${WRK_DIR}/Release-iphoneos/${FMK_RESOURCE_BUNDLE}.bundle" "${PRODUCTS_DIR}" || true
```

Comment:
@ElektrojungeMS @clpolet The error ignored by `|| true`, so it was not an error, just an message.
Personally I dislike the check of module name. I suggest the next options to replace it:
- Check if bundle exists
- Silent output

Reply:
I agree, my vote goes for "Check if bundle exists" since having a bundle named `${FMK_RESOURCE_BUNDLE}.bundle` seems to be conventional.